omniscidb/ArrowStorage/ArrowStorage.cpp

```diff
@@ -187,6 +187,53 @@ std::unique_ptr<AbstractDataToken> ArrowStorage::getZeroCopyBufferMemory(
   return nullptr;
 }
 
+std::unique_ptr<AbstractDataToken> ArrowStorage::getZeroCopyColumnData(
+    const ColumnRef& col_ref) {
```

Comment:
Almost copy-paste of `getZeroCopyBufferMemory`. Should we unite this methods, or add private utility method?

Reply:
Marking it as temporary solution, should be removed in further prs.